test-aether-integration-module/conf/pip/primary-requirements.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2018 by eHealth Africa : http://www.eHealthAfrica.org
+#
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+cat <<EOF
+######################################################################################################
+#                                                                                                    #
+#                             Primary requirements                                                   #
+#                                                                                                    #
+#                                 CHANGE ME !!!                                                      #
+#                                                                                                    #
+# Update me and then run 'docker-compose -f docker-compose-test.yml run integration-test pip_freeze' #
+#                                                                                                    #
+######################################################################################################
+
+
+aet.consumer>=0.9.1
+aether.client>=$( cat "VERSION")
```

Comment:
I didn't see the VERSION file in the volumes definition but if you say it works... nevermind this is not the purpose or this PR and any further problem can be fixed later.